lib/cookies.js

```diff
@@ -50,9 +51,9 @@ Cookies.prototype = {
     index = this.keys.index(data, remote)
 
     if (index < 0) {
-      this.set(sigName, null, {path: "/", signed: false })
+      if (autoSet) this.set(sigName, null, {path: "/", signed: false })
```

Comment:
i believe this would only happen if a cookie exists, but it is not signed correctly. thus, i don't think this would be set after every request unless the client is repeatedly sending invalid cookie values